pallets/liquidity-pools/src/message.rs

```diff
@@ -1044,7 +994,7 @@ mod tests {
 				decimals: 15,
 				restriction_set: 1,
 			},
-			"040000000000000001811acd5b3f17c06841c7e41e9e04cb1b536f6d65204e616d65000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000053594d424f4c00000000000000000000000000000000000000000000000000000f01",
+			"0b0000000000000001811acd5b3f17c06841c7e41e9e04cb1b536f6d65204e616d65000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000053594d424f4c00000000000000000000000000000000000000000000000000000f01",
```

Comment:
NIT NIT suggestion:

Maybe this super obscure line becomes clear using the `concat!` macro: i.e:

```
concat!(
    "0b" //msg_id
    "0000000000000001", //pool_id
    "811acd5b3f17c06841c7e41e9e04cb1b536f6d65204e616d65", // tranche_id
    "00000000000000000000000000000000", //...
    ..
)
```

And becomes easier also to create new tests